### PR TITLE
Initial version of `keybase wallet send`

### PR DESCRIPTION
**Pull request by patrickxb**
Note:

1. only sends to stellar addresses
2. plenty of more improvements before this is ever released

### CODE REVIEW

File: go/protocol/keybase1/extras.go [hunk 1]
```diff
@@ -2362,11 +2362,24 @@ func (s StellarBundle) CheckInvariants() error {
...
+	}
+	return StellarEntry{}, errors.New("primary stellar account not found")
+}
+
 func (a StellarAccountID) String() string {
 	return string(a)
 }
 
 func (s StellarSecretKey) String() string {
+	return "DONOTLOGDONOTLOGDONOTLOGDONOTLOGDONOTLOGDONOTLOGDONOTLOG"
```
Comment (mlsteele) [inline go/protocol/keybase1/extras.go]: `return "[redacted stellar secret key]"`

File: go/service/wallet.go [hunk 2]
```diff
@@ -47,27 +54,75 @@ func (h *walletHandler) WalletInit(ctx context.Context) (err error) {
...
+	ctx = h.logTag(ctx)
+	defer h.G().CTraceTimed(ctx, "WalletDump", func() error { return err })()
+	err = h.assertLoggedIn(ctx)
+	if err != nil {
+		return dump, err
+	}
+
+	// verify passphrase
+	username := h.G().GetEnv().GetUsername().String()
+	h.G().Log.Debug("resetting account for %s", username)
```
Comment (mlsteele) [inline go/service/wallet.go]: resetting account?

File: go/stellar/stellarsvc/service.go [hunk 3]
```diff
@@ -15,3 +20,175 @@ import (
...
+type balancesResult struct {
+	Status   libkb.AppStatus    `json:"status"`
+	Balances []stellar1.Balance `json:"balances"`
+}
+
+func (b *balancesResult) GetAppStatus() *libkb.AppStatus {
+	return &b.Status
+}
+
+func Balances(ctx context.Context, g *libkb.GlobalContext, accountID stellar1.AccountID) ([]stellar1.Balance, error) {
```
Comment (mlsteele) [inline go/stellar/stellarsvc/service.go]: this could live in `stellar/remote`, since it's exactly an API server call

**Comment by mlsteele** [comment]
Sorry, I caused some merge conflicts. Move `CheckInvariants` to `stellar1` extras and type `keybase1.StellarAccountID` -> `stellar1.AccountID`

File: go/service/wallet.go (hunk 2 above)
Comment (patrickxb) [reply go/service/wallet.go]: copy paste...thanks, will clean up.

**Comment by mlsteele** [comment]
Btw I think it would be great it `stellarsvc` were very simple and just had a list of jump functions that call into `stellar` and maybe pack and unpack arguments from rpc form. But not gonna hold up this PR for that opinion.

**Pull request merged**